### DISCUSSION TITLE
MSP Arm on an Rx PWM Output

### DISCUSSION THREAD

**Discussion by CREASE-gum-EAR** [ExpressLRS/ExpressLRS#2102] [Ideas]
Can an output on PWM receivers be set to output the MSP Arm when ExpressLRS is armed? This would let you use DJI and other HD Vtx systems without a flight controller or using a board that emulates MSP Arm. As you know, DJI Vtx only goes to high power mode when MSP Arm is set and can also start auto recording. Walksnail and HD Zero can use this feature too. This would be a great option when you aren't using a flight controller. I imagine in the web interface you could select MSP Arm in the drop down for one of the PWM outputs. Also, a PWM and +5v could be used to power a 1S Vtx by walksnail or HD Zero.

**Comment by mmosca** [comment]
You need both tx and rx connections (2 pins) to let the vtx know you are armed. 

I think what you want can probably be better served by dedicated hardware. 

Any microcontroller with a uart can fake being a flight controller for a vtx.
